app/forms/support_interface/application_forms/pick_course_form.rb

```diff
@@ -82,6 +96,18 @@ def courses
           .where(code: sanitize(course_code))
       end
 
+      def courses_for_provider(provider)
+        return [] if provider.blank?
+
+        @courses ||= Course
```

Comment:
Can remove the memoization here - only calling the method once, and if we did want to memoize we'd use a different pattern so that different provider values would return the correct results.

Reply:
Yea good shout